boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts

```diff
@@ -19,6 +19,8 @@
 		led1 = &blue_led;
 		led2 = &red_led;
 		usart-0 = &flexcomm0;
+		/* For pwm test suites */
+		pwm-0 = &sc_timer;
```

Comment:
Can the LEDs on this board be driven by a PWM? If so, please add pwm-led aliases to enable `samples/basic/rgb_led` and `samples/drivers/led_pwm`